doc/migrations/6.3.1_6.x.y.md

```diff
@@ -0,0 +1,13 @@
+## Change of the default TLS library used for C2S connections
+
+As of this release usage of `fast_tls` for Client to Server connections (C2S) has been deprecated.
+From now on the default TLS library for C2S is `just_tls`, which uses TLS implementation from Erlang OTP.
+In out performance tests `just_tls` is as performant as `fast_tls` and also has better standards compliance.
+
+It is still possible to use `fast_tls` for C2S, however `fast_tls` may be removed in some future release.
```

Comment:
Maybe we could emphasize that it **will** be removed in a future release? Actually, we are planning to remove it in the next one...